lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -509,15 +509,14 @@ describe('CardViewTextItemComponent', () => {
             expect(cardViewUpdateService.update).toHaveBeenCalledWith(property, 'updated-value');
         });
 
-        it('should NOT trigger the update event if the editedValue is invalid', async () => {
+        it('should trigger the update event if the editedValue', async () => {
             const cardViewUpdateService = TestBed.inject(CardViewUpdateService);
             spyOn(cardViewUpdateService, 'update');
-            component.property.isValid = () => false;
 
             updateTextField(component.property.key, 'updated-value');
             await fixture.whenStable();
 
-            expect(cardViewUpdateService.update).not.toHaveBeenCalled();
+            expect(cardViewUpdateService.update).toHaveBeenCalled();
         });
```

Comment:
Maybe there should be two tests:
- was `update` called when `property.isValid` returned `true`,
- was `update` called when `property.isValid` returned `false`.

Just to check for false positives as well.